CodeModel.Tests/Unit/CodeParser/CohesionAnalyzerTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using CodeModel.CodeParsers.CSharp.Exceptions;
+using CodeModel.Tests.DataFactories;
+using Xunit;
+using CodeModel.CodeParsers.CSharp.ProjectAnalyzer;
+
+namespace CodeModel.Tests.Unit.CodeParser
+{
+    public class CohesionAnalyzerTests
+    {
+        private static readonly CodeFactory TestDataFactory = new();
+
+        [Fact]
+        public void Test_Cohesion_Analyzer()
```

Comment:
Meaningful test name - when the test fail I should ideally know exactly which function is misbehaving and in which case.